volumes/rescu/README.md

```diff
@@ -1,45 +1,347 @@
+server name, schema, database, version, admin, 
+
+
+
+## Licenses placed on the data (Privacy concerns)
+> provide the link to thye known licences (e.g. CC)
+
+> if it's not a known licence, list the concerns and limitations due to the licensing
+
+
+
+## Limitation of the data
+>Data gaps make data unreliable for data requests.
+
+
+
+<!---## Links to any use the data: 
+>TBA--->
+
+
+
+<!---## Links to other publicly accessible locations of the data: 
+>TBA
+
+
+
+## Links/relationships to supplimentary data sets:
+> like centreline for studies/collisions
+>TBA
+
+
+
+
+## Do you expect citation for this dataset: 
+> TBA--->
+
+
+
+
+<!---# DATA & FILE OVERVIEW
+## List files
+> if the dataset or a version of it is stored in other places, list them here and provide information versioning information
+
+
+
+## Relevant datasets
+> if to work with the data, we need to access another dataset/ or there are relevant dataset, list them here with a link (schema/ folder address)
+
+
+
+## Format of accessible data
+> the final ready format 
+> what would be the first and easy way to access disaggregated data (low effort for us)
+
+
+
+## Available formats 
+> what format the data was collected in (original format)?
+> other formats that data is available: map, txt, geojson, excel, shapefile
+
+
+
+## Versions of the data
+> if yes, 
+1. name of file(s) that was updated: 
+2. Why was the file updated? 
+3. When was the file updated? 
+
+> use a table formatting (version id, created at, changes)
+--->
+
+
+# METHODOLOGICAL INFORMATION
+## **Description of methods used for collection/generation of data:**
+
+### 1- How is the data collected?	
+> Loop detectors installed on the ground of the road surface.
+Radar detectors (which function the same way as loop detectors) placed on the roadside. 
+
+### 2- What are the properties of the data? 	
+>Volume data at specific highway locations
+
+### 3- Is there an up to date map for this data? If yes, who should be updating the map?      
+>- A new map by D&A is underway
+>- Electrical contractor provides data feed which included latitude and longitude, however there are locational errors. The overall accuracy of the spatial data provided is questionable. 
+>- TPIM (Traffic Plant / Installation and Maintenance) would be responsible for updating this map as it falls within their purview. 
+### 4- How often is the data updated?	
+>Raw data feeds into oracle every 20 seconds. Data (Jim's aggregation) pushed into Oracle daily. Data Pipeline Flow and Transformation
+## **Methods for processing the data: Data Pipeline Flow and Transformation- How does the data move/transform through the organization?**
+	
+### 1- How is the data organized and aggregated? 	
+> Raw data is recorded in 20 second increments. Data handled by D&A is aggregated to 15 minute increments and produced as reports out of Oracle. Data is matched to arterycode (directional).Loop detectors, radar detectors have coordinates. 
+
```

Comment:
Need a space between `(directional).` and `Loop`

Reply:
Added.